src/PostgREST/PgVersions.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass #-}
+module PostgREST.PgVersions
```

Comment:
Let's move this to `PostgREST.DbStructure.PgVersion` for now.

Reply:
Ok